lib/price_calculator.rb

```diff
@@ -0,0 +1,2 @@
+class Price_calculator
```

Comment:
Style/ClassAndModuleCamelCase: Use CamelCase for classes and modules. (https://github.com/bbatsov/ruby-style-guide#camelcase-classes)